sklearn/utils/estimator_checks.py

```diff
@@ -2251,8 +2331,7 @@ def check_classifiers_predictions(X, y, name, classifier_orig,
                                (classifier, ", ".join(map(str, y_exp)),
                                 ", ".join(map(str, y_pred))))
 
-    # training set performance
-    if name != "ComplementNB":
+    if not api_only and name != "ComplementNB":
```

Comment:
@NicolasHug we are missing this check in the `api_only` PR. However, there is something fishy here because either I am passing `api_only=True`, this `if` statement is run. I will investigate

Reply:
It's because `api_only` isn't passed in the calls to `check_classifiers_predictions`. I'll update the other PR, thanks for noting

Reply:
We did not pass `api_only` in the other when calling this function in `check_classifiers_classes`.

Reply:
I was 16 seconds too slow :)